### ISSUE TITLE
New computer

### ISSUE THREAD

**Issue by ciadonisi** [fdac21/students#95]
I just got a new computer over the weekend. Can I still use the same port number? Also, do I need to send you my new pubkey?

**Comment by audrism** [comment]
yes, port is the same. You can simply cut and paste new public key to .ssh/authorized_keys file in the container
